napalm_ios/ios.py

```diff
@@ -354,6 +354,8 @@ def commit_config(self):
                 raise ReplaceConfigException("Candidate config file does not exist")
             if self.auto_rollback_on_error:
                 cmd = 'configure replace {} force revert trigger error'.format(cfg_file)
+            elif confirmed:
+                cmd = 'configure replace bootflash:{} force time {}'.format(cfg_file, confirmed)
```

Comment:
Netmiko should automatically try to autodectect by doing a `dir`. If this fails, however, you can still set it using NAPALM optional_args.

We should create an issue, however, if you are seeing this fail.